internal/matchers/archive.go

```diff
@@ -84,3 +84,10 @@ func Rar(in []byte) bool {
 func Warc(in []byte) bool {
 	return bytes.HasPrefix(in, []byte("WARC/"))
 }
+
+// Zstd matches a Zstandard archive file
+func Zstd(in []byte) bool {
+	return (len(in) >= 4) &&
```

Comment:
This return can have the same meaning while deleting some of the parentheses. No need for additional parentheses unless they help the reader reason about the order of evaluation.